app/controllers/api/v1/customers/random_customers_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::Customers::RandomCustomersController < ApplicationController
```

Comment:
since we are namespacing under `Customers` - we probably don't need to name this controller `RandomCustomersController` - `RandomController` is probably sufficient